account_invoice_supplierinfo_update_on_validate/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================================================
+Update Supplier Info of product from Supplier Invoice Validation
+================================================================
+
+This module is a glue module installed if the following module are installed:
```

Comment:
Hi. I understand the aim of this module. (make supplier update when invoice is validated), but I don't understand very well the description. For me a glue module is a module auto_installed, when 2 (or more) modules are installed. That is not the case here.
Don't you think you should update this description ?
(not a blocking point).